Tests/FluentAssertions.Equivalency.Specs/SelectionRulesSpecs.cs

```diff
@@ -1489,5 +1490,551 @@ public void Including_an_interface_property_through_inheritance_should_work()
                 .Including(a => a.Value2)
                 .RespectingRuntimeTypes());
         }
+
+        [Fact]
+        public void When_browsable_field_differs_including_non_browsable_members_should_not_affect_result()
```

Comment:
🤔 Why would including non browsable members effect the test at all? It will throw regardless and your assertion is too generic to see the difference.
⛏️ Since you can't use commas in test names, add `then` to make it readable, e.g.  `When_browsable_field_differs_then_including_non_browsable_members_should_not_affect_result`

Same comment on all other similar tests

Reply:
Yeah I think I'll just remove the `IncludingNonBrowsableMembers` method. All it does is undoes the flag that `ExcludingNonBrowsableMembers` sets. In retrospect, I don't really see a use case for this.